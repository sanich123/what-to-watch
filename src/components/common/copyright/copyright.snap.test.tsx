import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import Copyright from './copyright';

describe('Copyright comonent', () => {
  it('should render correctly', () => {
    const {container} = render(<Copyright />);
    expect(container).toMatchSnapshot();
  });
});
