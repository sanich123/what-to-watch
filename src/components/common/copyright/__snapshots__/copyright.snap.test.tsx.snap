// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright comonent should render correctly 1`] = `
<div>
  <div
    class="copyright"
  >
    <p>
      Â© 2019 What to watch Ltd.
    </p>
  </div>
</div>
`;
